tests/integration/queries/tpch/query_3.py

```diff
@@ -0,0 +1,34 @@
+import pytest
+from ....common import get_substrait_plan
```

Comment:
Hmm.... may be once we are cleared with the functionality and addressing majority of the concerns. It would be better to package this. Include a `setup.py` and other required components. Then we can release the functional consumer test suite nightly and a stable release in matched with Substrait, Arrow, Duckdb, ... releases. 


cc @gforsyth @jacques-n @rok @jvanstraten